CHANGELOG.md

```diff
@@ -50,7 +50,7 @@ The first release of the tool refactored and reimplemented in React and TypeScri
 - [MNT] Added post-merge hook and updated scripts [#38](https://github.com/neurobagel/query-tool/pull/38) ([@Sauradip07](https://github.com/Sauradip07))
 - [CI] Set up GitHub pages deployment CI [#50](https://github.com/neurobagel/query-tool/pull/50) ([@rmanaem](https://github.com/rmanaem))
 - [CI] Set up ESLint CI [#47](https://github.com/neurobagel/query-tool/pull/47) ([@rmanaem](https://github.com/rmanaem))
-- [CI]  Implement Prettier CI Check for Code Consistency [#44](https://github.com/neurobagel/query-tool/pull/44) ([@samadpls](https://github.com/samadpls))
+- [CI] Implement Prettier CI Check for Code Consistency [#44](https://github.com/neurobagel/query-tool/pull/44) ([@samadpls](https://github.com/samadpls))
```

Comment:
This is an autogenerated file. Could you please roll this back - I added it to the prettierignore file in #83